utils/build/docker/dotnet/parametric/Endpoints/ApmTestApiOtel.cs

```diff
@@ -112,12 +113,56 @@ private static async Task<string> OtelStartSpan(HttpRequest request)
             }
         }
         
+        var linksList = new List<ActivityLink>();
+
+        if (requestBodyObject.TryGetValue("links", out var links))
+        {
+            foreach (var spanLink in (dynamic)links)
```

Comment:
Let's avoiding using `dynamic` here since we would avoid doing so in the tracer library. I think that you can cast `links` to `Newtonsoft.Json.Linq.JArray` and the typing should all work out, but let me know if that doesn't

Reply:
Addressed in the latest commit. [a0364b1](https://github.com/DataDog/system-tests/pull/2543/commits/a0364b14416f55ac5483456ba82d4ea5d26aa865)